src/index.js

```diff
@@ -71,16 +73,26 @@ const checkSignInStatus = () => {
   }
 }
 
+// const pullCustmerSearchValue = () => {
+//   $('.date-input').val() =
+//
+// }
+
 const instanciateCustomer = id => {
   let selectedUser = allData.userData.find(user => {
     if (user.id === id) {
       return user;
     }
   })
-  signedInUser = new Customer(selectedUser.id, selectedUser.name, allData, getTodaysDate())
+  signedInUser = new Customer(selectedUser.id, selectedUser.name, getTodaysDate())
   signedInUser.findAllBookings()
 }
 
+export const instanciateHotel = () => {
```

Comment:
You misspelled instantiate.

Reply:
Good eye @pJanks! I'll correct that immediately.